cmd/scollector/collectors/stream.go

```diff
@@ -0,0 +1,82 @@
+package collectors
+
+import (
+	"reflect"
+	"runtime"
+	"sync"
+	"time"
+
+	"bosun.org/collect"
+	"bosun.org/metadata"
+	"bosun.org/opentsdb"
+)
+
+type StreamCollector struct {
```

Comment:
You could just do `collect.Add("localListener.relayed", tags, len(mdp))` or similar in your relay function as you see datapoints.